ui/library/less/styles.less

```diff
@@ -24,8 +15,26 @@
 }
 
 .reader__page-overlay__bounding-box-highlighted {
-  background-color: @pdflib-highlight-background;
-  opacity: .2;
+  background-color: @primary-blue;
+  mix-blend-mode: screen;
+
+  &:hover {
+    background-color: @cornflower-blue;
+    mix-blend-mode: screen;
+  }
+
+  &.selected {
+    background-color: @cornflower-blue;
+    mix-blend-mode: screen;
+  }
```

Comment:
Since `&:hover` and `&.selected` have the same css rules, I think you can merge them and write as `&:hover, &.selected`. Also the `mix-blend-mode: screen` is defined outside as well, do you still have to write the rule inside?

Reply:
@yensung  that is true i can combine them together for a more simple code

Reply:
Agreed, combining the hover and selected states and removing the duplicate `mix-blend-mode`s would make this cleaner 👌